test/test_SW_SoilWater.cc

```diff
@@ -53,82 +50,71 @@ namespace{
   TEST(SWSoilWaterTest, SWSWCSdjustSnow){
     // setup mock variables
     SW_Site.TminAccu2 = 0;
-    RealD temp_min = 0;
-    RealD temp_max = 10;
-    RealD ppt = 1;
-    RealD rain = 1.5;
-    RealD snow = 1.5;
-    RealD snowmelt = 1.2;
-    // test 1, since TminAccu2 is < temp_ave, we expect SnowAccu to be 0 and thus rain is ppt - SnowAccu
+    RealD temp_min = 0, temp_max = 10, ppt = 1, rain = 1.5, snow = 1.5,
+    snowmelt = 1.2;
+
+    // test 1, since TminAccu2 is < temp_ave, we expect SnowAccu to be 0 and thus
+    // rain is ppt - SnowAccu
     SW_SWC_adjust_snow(temp_min, temp_max, ppt, &rain, &snow, &snowmelt);
     EXPECT_EQ(rain, 1);
     EXPECT_EQ(snow, 0);
-
     // Reset to previous global states
     Reset_SOILWAT2_after_UnitTest();
   }
 
   // Test the 'SW_SoilWater' function 'SW_SWCbulk2SWPmatric'
   TEST(SWSoilWaterTest, SWSWCbulk2SWPmatric){
```

Comment:
Add one more test for when fractiongravel = 1. Any if...else statement in a funtcion should trigger a unit test.